### PR TITLE
Fix Home page

### PR DESCRIPTION
**Pull request by sidelong44**
Fixed "Recent activities" heading
Fixed left/right arrow buttons on home page
Dark bg for hot-lists/radio stations on home page

### CODE REVIEW

File: play-music.user.css [hunk 1]
```diff
@@ -241,7 +244,6 @@ gpm-row-item:not(:last-child) .underline.gpm-row-item { border-bottom-color: #16
 gpm-row-item > a { color: rgba(255,255,255,0.8)!important }
 gpm-vertical-list #items.gpm-vertical-list > *:hover, gpm-vertical-list #items.gpm-vertical-list > *[focused] { background: #282828; }
 gpm-card-grid > [slot="title"] { color: white; }
-sj-card #details.sj-card > [slot="title"] { color: white; }
```
Comment (sidelong44) [inline play-music.user.css:244]: Moved higher

**Comment by sidelong44** [comment]
@macfeaster, this is another fix. Changes for home page.
I also bumped the version of style.

**Comment by macfeaster** [comment]
Seems good to me, this is mostly just modifying existing rules so they apply under more conditions, right? Could you provide me with before/after screenshots on the arrows + hot lists? Then we can merge 🎉

**Comment by sidelong44** [comment]
@macfeaster, of course! Here are the differences:
Perhaps Google is migrating the site to Web-components, which is why such problems appear...

---

Home top before:
image
Home top after:
image

Home hot-lists/radio stations before:
image
Home hot-lists/radio stations after:
image

**Comment by macfeaster** [comment]
Perfect, thank you so much! This is correct styling indeed, merging!

**Pull request merged**